.gitignore

```diff
@@ -1,6 +1,6 @@
 Cargo.lock
 tcli.log
-./*/
+/.*/
```

Comment:
what does that actually exclude?

Reply:
I think this was removed from main

Reply:
As this was falsely added

Reply:
It excludes all directories starting with . that are in the root directory

Reply:
And I put it it here on purpose, because the previous `./*/` was a typo